src/engine/effects/engineeffectchain.cpp

```diff
@@ -137,19 +137,32 @@ bool EngineEffectChain::enableForGroup(const QString& group) {
 }
 
 bool EngineEffectChain::disableForGroup(const QString& group) {
-    GroupStatus& status = m_groupStatus[group];
+    GroupStatus& status = getGroupStatus(group);
     if (status.enable_state != EffectProcessor::DISABLED) {
         status.enable_state = EffectProcessor::DISABLING;
     }
     return true;
 }
 
+EngineEffectChain::GroupStatus& EngineEffectChain::getGroupStatus(const QString& group) {
+    for (QLinkedList<GroupStatus>::iterator it = m_groupStatus.begin(),
+                 end = m_groupStatus.end(); it != end; ++it) {
+        GroupStatus& status = *it;
+        if (status.group == group) {
+            return status;
+        }
+    }
+    GroupStatus status(group);
+    m_groupStatus.append(status);
+    return m_groupStatus.last();
```

Comment:
This method returns a reference and status is a local variable -> will be invalid once the method returns.